appveyor.yml

```diff
@@ -16,5 +16,5 @@ platform:
 test_script:
   - ruby --version
   - gem --version
-  - bundler --version
+  - bundle --version
```

Comment:
Strange.. both should've been working..

Reply:
`bundler` used to work so I'm guessing there was a change on AppVeyer.

Reply:
Yep, I got them to push a fix.. testing now..

Reply:
You can remove this change.. its not needed anymore :smiley:

Reply:
Restarting the failed tests should pass now..

Reply:
ok, what the chance they will make the same mistake again I wonder.

Reply:
Its actually a bug in Rubygems.. The lead maintainer at AppVeyor has an issue opened at Bundler as well.. considering the fact that he shipped a quick-fix, the chances of this occurring again are slim yet not zero..